### DISCUSSION TITLE
Need help with docker in debian

### DISCUSSION THREAD

**Discussion by derwegi** [achannarasappa/ticker#182] [Q&A]
I'm new to docker and want to run the ticker on my debian machine that has docker installed. When I run the command like in the readme:

`docker run -it --rm achannarasappa/ticker`

the program runs but what to do next? I see a black terminal with no content but in the bottom "ticker q: exit scroll up and down." How can I configure my stocks? Or how do I access the configure file?

**Comment by rinpoche** [comment]
Not sure if you have resolved this issue, but do this: 
create a file _ticker.yaml_ in your current directory (use the sample ticker.yaml from the code page of the repo), and run
**docker run -it -v (pwd)/ticker.yaml:/.ticker.yaml --rm achannarasappa/ticker**

I using Fish shell hence (pwd), you can try $(pwd)
